packages/web-components/src/components/ic-badge/ic-badge.tsx

```diff
@@ -42,6 +42,11 @@ export class Badge {
    */
   @Prop() accessibleLabel?: string;
 
+  /**
+   * The id of the badge component.
+   */
+  @Prop() badgeId: string = "badge";
```

Comment:
This doesn't resolve the ticket's core issue of having duplicate IDs on badges. If ICDS customers don't change their code, they'd still have duplicate IDs of "badge" on their page.